include/spdlog/details/os-inl.h

```diff
@@ -567,7 +567,7 @@ SPDLOG_INLINE filename_t dir_name(const filename_t &path) {
     #pragma warning(disable : 4996)
 #endif  // _MSC_VER
 std::string SPDLOG_INLINE getenv(const char *field) {
-#if defined(_MSC_VER) && defined(__cplusplus_winrt)
+#if defined(_MSC_VER) && (WINAPI_FAMILY != WINAPI_FAMILY_DESKTOP_APP)
```

Comment:
Better is check if WINAPI_FAMILY defined:

#if defined(_MSC_VER) && defined(WINAPI_FAMILY) && (WINAPI_FAMILY != WINAPI_FAMILY_DESKTOP_APP)
// non-desktop MSVC code
#endif

Reply:
Hmm should we also check if WINAPI_FAMILY_DESKTOP_APP is defined then?

Reply:
Unclear. I cannot find any official doc aside from [this page](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/migrate-to-windows-app-sdk/misc-info). That being said, it's unlikely one macro would be defined but not the other.

Reply:
I guess checking them both is the safe thing